ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+*** Settings ***
```

Comment:
this probably should go under Model Serving suite. What do you think? cc: @jgarciao This suite was implemented for RHODS overall monitoring, should we start moving here the monitoring/metrics tests related to model serving as well?

Reply:
If you want to move this suite under model serving it's fine for me, as the 200__monitor_and_manage folder is focused on SRE Monitoring as Berto said